CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [0.18.5](https://github.com/eclipse/lemminx/milestone/31?closed=1) (February 14, 2022)
+
+### Enhancements
+
+- Prevent suspicious directory traversal. See [#1171](https://github.com/eclipse/lemminx/pull/1171).
```

Comment:
I would put this under Bug Fixes as its less of a change visible to users. Also for consistency, let's use `*` for the entries as opposed to `-`